sentry-samples/sentry-samples-android/src/main/AndroidManifest.xml

```diff
@@ -109,7 +113,7 @@
         <meta-data android:name="io.sentry.traces.activity.auto-finish.enable" android:value="false" />
 
         <!-- how to enable the UI auto instrumentation for tracing -->
-        <meta-data android:name="io.sentry.traces.user-interaction.enable" android:value="true" />
+<!--        <meta-data android:name="io.sentry.traces.user-interaction.enable" android:value="true" />-->
```

Comment:
Can you elaborate on:

> Disabled auto user interaction instrumentation (creates concurrent transactions not currently supported by profiling)

I don't follow, there should be a single transaction in the scope, this is like this by design, so I'm not sure what you mean by concurrent transactions.
Also, disabling another feature isn't the best idea, either when testing this feature, we manually disable and enable it again (not coming the changes).
We can also create a new flavor that enables and disables certain features.
We can also create a new sample app for profiling as suggested before as well if needed.

Reply:
The thing is that when i added an activity where i click on a button and it starts a transaction to be profiled.
But when i click on the button an automatic transaction is started first, so that one is profiled. When the manual transaction is started, the automatic one hasn't finished, yet, so the profiler ignores the manual transaction, since it doesn't have the notion of scope.
We're currently working on allowing profiling to work in these cases.
We can reenable the user-interaction option, disabling it when testing the profiler.
I'm not sure a new sample app just for profiling is worth it, as we can find some edge cases we didn't consider when using other options of the sdk, too.

Reply:
Is it possible this uncovered a scenario we need to address? Should the old transaction be completed (and so profiling ended) when the new starts? Or should the new transaction not start because we already have one on going. We only create ui event transactions if none exists?

Reply:
We create it only if none exists, but by default it's automatically stopped after 3 seconds. This case is a transaction manually started in those 3 seconds.
we discussed this scenario, and we decided to make profiling work with multiple transactions [here](https://www.notion.so/sentry/Profiling-concurrent-transactions-37da1625332e40799dc0e0bae90ced63).
We should support it anyway, as this has less attrition for the user, and it's more important in the backend clients, where it's easy to have multiple concurrent transactions, possibly spawn over multiple threads.

Reply:
I'd say let's move ahead with this then and we can track this on an issue? Would you please open an issue?

Reply:
yeah, we already created a [doc](https://www.notion.so/sentry/Profiling-concurrent-transactions-37da1625332e40799dc0e0bae90ced63)
Anyway, i just created [this issue](https://github.com/getsentry/sentry-java/issues/2209) and i'll track it in the new pr i'll do